### PR TITLE
Add mac structs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
With this one, I'll have almost all the structs I need directly in libc. :)

### CODE REVIEW

**Comment by alexcrichton** [comment]
Looks like CI is failing

**Comment by GuillaumeGomez** [comment]
@alexcrichton: Fixed!

File: src/unix/bsd/apple/mod.rs [hunk 1]
```diff
@@ -22,6 +22,13 @@ pub type nl_item = ::c_int;
 pub type id_t = ::c_uint;
 pub type sem_t = ::c_int;
 pub type idtype_t = ::c_uint;
+cfg_if! {
```
Comment (alexcrichton) [inline src/unix/bsd/apple/mod.rs]: There are submodules to do the x86/x86_64 distinction, can this be pushed down into those submodules?

Comment (GuillaumeGomez) [inline src/unix/bsd/apple/mod.rs]: Sure.

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by GuillaumeGomez** [comment]
@alexcrichton: Can you make a new release please?

**Comment by alexcrichton** [comment]
The README [details information on that](https://github.com/rust-lang/libc#releasing-your-change-to-cratesio)

**Pull request merged**